webrob/pages/main.py

```diff
@@ -28,59 +30,78 @@ def track_login(sender, user, **extra):
     session['username'] = user.username
     session['api_token'] = user.api_token
 
+
 @user_logged_out.connect_via(app)
 def track_logout(sender, user, **extra):
     if 'user_container_name' in session:
         docker_interface.stop_user_container(session['username'])
         session.pop('user_container_name')
 
+
 @app.errorhandler(Exception)
 def redirect_unhandled_exception(e):
     app.logger.error('Unhandled Exception: %s', (e))
     app.logger.error(traceback.format_exc())
     # TODO: show an appology page to the user
     return redirect(url_for('user.login'))
 
+
+# method to send woff font files from node_modules
+@app.route('/user/<node_modules>/<typeface_oswald>/<files>/<path:file_name>')
+@app.route('/<node_modules>/<typeface_oswald>/<files>/<path:file_name>')
+def send_woff_file_from_neem_directory(node_modules, typeface_oswald, files, file_name):
+    app.logger.info("Send_woff_file_from_neem_directory: " + file_name)
+    file_path = NODE_MODULES_STORAGE_PATH_TAG + node_modules + "/" + typeface_oswald + "/" + files + "/"
```

Comment:
best practice would be to use *os.path.join* to build paths, then you do not need to spell out the Linux-specific "/"